src/client.rs

```diff
@@ -303,14 +312,13 @@ mod tests {
         }"#
         );
 
-
         test_get(
             &mock_server,
-            Client::create_with_server_collection(
-                &mock_server_address,
-                "url-classifier-skip-urls",
-                Some(Box::new(VerifierWithVerificatonError {})),
-            ),
+            Client::builder()
+            .server_url(&mock_server_address)
+            .collection_name("url-classifier-skip-urls")
+            .verifier(Box::new(VerifierWithVerificatonError {}))
+            .build(),
```

Comment:
Any idea why this one is not indented like the others?

Reply:
oops, fixed in latest commit